lib/src/prefabs/files/fileCard/FileCard.js

```diff
@@ -10,6 +10,7 @@ function FileCard({
   insetMeta,
   children,
   size,
+  innerClassName,
   ...cardProps
```

Comment:
I'm not too sure if we need to add this prop for `fileCard`.

`...cardProps` is collected in this pattern because the `Card` component has the `innerClassName` prop. File card is just built on top so do we need to add this in?

If you're passing inner class name then it'l be spread to the card component too.

------

This is similar to other areas that do this like proxy components like `Input` and `Buttons`.